### DISCUSSION TITLE
help!!!

### DISCUSSION THREAD

**Discussion by Emphantom4d** [feelfreelinux/octo4a#358] [Q&A]
hello. samsung s22 ultra app crash. no work.

**Comment by Emphantom4d** [comment]
![Screenshot_20230125_141032_OctoPrint For Android](https://user-images.githubusercontent.com/65423019/214560123-6f2ee858-233d-45a9-8dfb-18515f829cd3.jpg)

**Reply by TechWithMikeD** [reply]
The app is broken right now. Nearly everyone is running into this when installing this version. You'll have to wait for a fix, it seems.
